src/main/java/org/monarchinitiative/fhir2hpo/codesystems/CodeContainer.java

```diff
@@ -11,50 +11,50 @@
 //TODO: Consider allowing jar user to add their own mappings as original library did
 public class CodeContainer {
 	
-	private static Map<String,String> fhirv2interpretations = Collections.unmodifiableMap(Stream.of(
-			new SimpleEntry<>("<", "L"),
-			new SimpleEntry<>(">", "H"),
-			new SimpleEntry<>("A", "A"),
-			new SimpleEntry<>("AA", "A"),
-			new SimpleEntry<>("AC", "POS"),
```

Comment:
Note in the transformation that I further reduced the internal codes using additional logic from the other library. "POS" now becomes "H", and "NEG" becomes "N". If there is a reason we need to maintain the "POS" and "NEG" internally, let me know.

Reply:
It just occurred to me that when I parse the annotations file, those codes may be "POS" or "NEG", so some additional work is needed.

Reply:
Hi Amy, it is possible to combine "Pos" and "H", "Neg" and "N", as positive outcome usually equals "high than normal" and negative outcome usually equals "lower than normal". We decided to separate them because it is more natural.

Reply:
updated comment: negative outcome usually equals "normal"